bus/bus.go

```diff
@@ -642,6 +655,7 @@ func New(s Syncer, cm ChainManager, tp TransactionPool, w Wallet, hdb HostDB, cs
 		"POST   /hosts/:hostkey": b.hostsPubkeyHandlerPOST,
 
 		"GET    /contracts":             b.contractsHandler,
+		"POST   /contracts/ancestors":   b.contractsAncestorsHandlerGet,
```

Comment:
it could be `GET /contracts/:id` with the `startheight` as a query string param (and thus optional)?

Reply:
the handler is named `Get` so I figure you originally had it as a `GET`

also (`nit`) we should be consistent and do `someHandlerGET`